packages/tpetra/core/src/Tpetra_Details_crsUtils.hpp

```diff
@@ -133,20 +132,26 @@ pad_crs_arrays(
 
     // Copy over indices
     {
-      auto indices_old_subview = subview(indices, range(this_row_beg, this_row_beg+used_this_row));
-      auto indices_new_subview = subview(indices_new, range(row_ptr_beg(i), row_ptr_beg(i)+used_this_row));
-      // just call memcpy; it works fine on device if this becomes a kernel
-      memcpy(indices_new_subview.data(), indices_old_subview.data(), used_this_row * sizeof(inds_value_type));
+      auto indices_old_subview = subview(indices,
+        range(this_row_beg, this_row_beg+used_this_row));
+      auto indices_new_subview = subview(indices_new,
+        range(row_ptr_beg(i), row_ptr_beg(i)+used_this_row));
+      using Kokkos::Experimental::local_deep_copy;
```

Comment:
I'm not excited about relying on Kokkos::Experimental.  
What is Kokkos' plan for local_deep_copy?

Reply:
@kddevin Christian tells me that it will go into the Kokkos namespace in the next Kokkos release.  There will also be team versions.

Reply:
OK; I'll add a Kokkos issue requesting the move and link it back to this issue.